PowerPointLabs/PowerPointLabs/TextCollection/CommonText.cs

```diff
@@ -3,12 +3,16 @@
     public class CommonText
     {
         # region Common Error
+        public const string ErrorCaption = "Error";
+        public const string ErrorSlideSelectionTitle = "Slide selection error";
+        public const string ErrorDuringSetup = "Error During Setup.";
```

Comment:
Inconsistent names. They are all error message titles right? So why is one "Caption", while the other "Title" and the other with nothing? 
Also, was "Caption" used in other parts of TextCollection? I don't really remember that word being used. The names we use should be consistent, so that it is easy for developers to understand. For example, we should use "Title" here.

Reply:
@IanTeo 

1. Refering to this script(https://github.com/junkiattan/PowerPointLabs/blob/12dbbd254f97140e0a4f3f07cadecf4bec8d222d/PowerPointLabs/PowerPointLabs/TextCollection/ShapesLabText.cs#L34) "Caption" was used.

2. In lines 35 and 36 of the same file, the error names of the strings had nothing while others had the words "Category". Furthermore, some strings are called Labels while others are called Titles

I apologise if I named the string inappropriately and I agree with you on the importance of naming the names consistently, but I have to say that the names of the variables were not that standardized or documented in the first place, at least not enough for me to follow a convention and name appropriately when I was converting all the message dialogs into string variables.

Reply:
I have changed ErrorCaption to ErrorTitle. However ErrorDuringSetup was following the error messages that I saw on most of the files in TextCollection so I did not change that

Reply:
No problem. Sorry that it was inconsistent in some parts, which made it hard for you to follow. Hopefully you can help make the code base more consistent through your time working on pptlabs :)